tests/integration/ledger_sync/ls_outboundq_tests.py

```diff
@@ -271,27 +284,40 @@ def test_add_role_member_outqueue():
         )
         result = assert_api_success(response)
         proposal_response = get_proposal_with_retry(session, result["proposal_id"])
-        proposal = assert_api_success(proposal_response)
+        assert_api_success(proposal_response)
         # Logging in as role owner
         credentials_payload = {
             "id": user1_payload["username"],
             "password": user1_payload["password"],
         }
         log_in(session, credentials_payload)
         # Approve proposal as role owner
-        approval_response = approve_proposal(session, result["proposal_id"])
-        end_depth = get_outbound_queue_depth()
-        assert end_depth > start_depth
+        approve_proposal(session, result["proposal_id"])
 
-        # Check status of new outbound_entry
+        # NOTE: members field contains an empty string because in NEXT
```

Comment:
why do we put anything in the outbound queue at all during NEXT only mode?

Reply:
Did we decided to not put anything in the outbound_queue if we are in NEXT only mode? If so, this can be changed so that NEXT-only items are not inserted into outbound_queue table.

Reply:
yeah, we're not going to insert anything into the outbound queue during NEXT only mode. Can you include this file and function in a note on an issue ticket to only insert to the outbound queue when **not** in NEXT-only mode (if one doesn't already exist)?